README.md

```diff
@@ -82,7 +82,7 @@ yarn deploy --network $NETWORK
 Verify on etherscan:
 
 ```
-npx hardhat verify --network $NETWORK DEPLOYED_CONTRACT_ADDRESS
+yarn hardhat verifyContracts --network $NETWORK
```

Comment:
The `hardhat` command is not defined in `package.json`:

```suggestion
npx hardhat verifyContracts --network $NETWORK
```

Reply:
? But it is exported in the hardhat config. I think this is sufficient, isn't it?

Reply:
Mmh, I thought `yarn hardhat` didn't exist, since we don't define the `hardhat` script in `package.json`, but apparently it's working anyway. Nothing to do here!